contracts/HubPool.sol

```diff
@@ -225,7 +247,7 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         uint64 poolRebalanceLeafCount,
         bytes32 poolRebalanceRoot,
         bytes32 destinationDistributionRoot
-    ) public onlyIfNoActiveRequest {
+    ) public noActiveRequests {
```

Comment:
As discussed IRL:
- consider adding a comment that after `initiateRelayerRefund` is called, if the `poolRebalanceRoot`, `destinationDistributionRoot`, `poolRebalanceLeafCount`, and  `bundleEvaluationBlockNumbers` set is disputable then this can be challenged. Once the challenge period passes, then the roots are no longer disputable, and only `executeRelayerRefund` can be called and `initiateRelayerRefund` can't be called again until all leafs are executed. 
- Examples of some characteristics of these roots that could be disputed: `poolRebalanceRoot.netSendAmount != destinationDistributionRoot.amountToReturn` for example

Reply:
Motivation is updated comments + interface for the spoke pool `distributeRelayerRefund` in my PR: https://github.com/across-protocol/across-smart-contracts-v2/pull/20

Reply:
will do!